core/primitives/Cargo.toml

```diff
@@ -23,7 +23,7 @@ rand = "0.7"
 reed-solomon-erasure = "4"
 jemallocator = { version = "0.3", optional = true }
 hex = "0.4"
-num-rational = "0.2.4"
+num-rational = { version = "0.2.4", features = ["serde"]}
 primitive-types = "0.7"
 
 borsh = "0.7.1"
```

Comment:
There are also 3 dependencies below that we'll have to publish first before we can publish this

Reply:
`near_crypto` seems publishable as is.
`near-primitives` and `near_vm_errors` depends on `near-rpc-error-macro`.  So it would make sense to bring `near_vm_errors` into `near_primitives`.
Leaving `near-rpc-error-macro` to be published, which itself depends on `near-rpc-error-core`, which seems publishable as is.

Reply:
It seems we should move `near-rpc-error-macro` to optional dependency and only use it when we need to regenerate error schema. @frol